IdentityCore/src/requests/sdk/msal/MSIDDefaultBrokerResponseHandler.m

```diff
@@ -199,4 +200,22 @@ - (NSError *)resultFromBrokerErrorResponse:(MSIDAADV2BrokerResponse *)errorRespo
     return brokerError;
 }
 
+- (BOOL)canHandleBrokerResponse:(NSURL *)response
```

Comment:
seems like a duplicate logic. Should it go to the base class and here you just call super with different params?

Reply:
done